packages/flutter_test/test/live_binding_test.dart

```diff
@@ -16,4 +18,93 @@ void main() {
     // starts with a PointerAddedEvent.
     await gesture.addPointer();
   });
+
+  binding.framePolicy = LiveTestWidgetsFlutterBindingFramePolicy.fullyLive;
```

Comment:
This won't work. Tests might be executed in random order, and the code "on top of them" isn't guaranteed to run inside the test itself.

You either have to set the frame policy in the test and have that just work, or create separate test files for different policies.

You _might_ get this working if you use `group`s.